docs/conversion/sakai_21_0-21_1_oracle_conversion.sql

```diff
@@ -47,6 +47,6 @@ CREATE INDEX IDX_FCI_STATUS ON FILE_CONVERSION_QUEUE(STATUS);
 -- END SAK-45575
 
 -- START SAK-45580
-DROP INDEX IDX_FCI_REF_TYPE ON FILE_CONVERSION_QUEUE;
+DROP INDEX IDX_FCI_REF_TYPE;
```

Comment:
This is OK

-- Se ha hecho borrando desde la interfaz gráfica
DROP INDEX SAKAI_PR.IDX_FCI_REF_TYPE;